backend/src/lambda/start-game.ts

```
@@ -10,17 +15,82 @@ const requestParameters = {
   gameCode: String,
 }
 
+let availableAssignments: PlayerModel[] = [];
+let lastPlayerAssignee: PlayerModel;
+let lastPlayerAssigned: PlayerModel | undefined;
+
 async function handler(event: APIGatewayEvent, context: Context) {
   const { authToken, gameCode } = validateRequestBody(event.body, requestParameters)
 
   // verify authtoken, retrieve user
+  let userId = await authenticate(authToken);
   // retrieve game by gameCode
+  let game = await getGame(gameCode);
   // retrieve host of game
+  let host = await getUser(game!.hostName);
+
   // verify host of game id == user.id
-  // make assignments and text/email to everyone
-  // update game status as started
+  if (host!.id == userId) {
```

Instead of continuing _if_ `id` matches, throw an error if id _doesn't_ match. The general flow of a handler should expect everything to work, with breaks when there is a problem:
```ts
if (host!.id != userId)
  throw new HTTPError(403, constants.strings.mustBeHostToStartGame)
...
```
(You'll have to add a string for that too)